ui/src/app/settings/containers/Repositories/Repositories.js

```diff
@@ -1,27 +1,26 @@
 import { connect } from "react-redux";
 import PropTypes from "prop-types";
-import React from "react";
+import React, { useEffect } from "react";
 
 import { RepositoryShape } from "../../proptypes";
 import actions from "../../actions";
 import selectors from "../../selectors";
 
-export class Repositories extends React.Component {
-  componentDidMount() {
-    this.props.fetchRepositories();
-  }
+export function Repositories(props) {
```

Comment:
as before, `export const Repositories = ({ fetchRepositories }) => {` is probably nicer.

Reply:
Done.